### ISSUE TITLE
documentation error: configuring eXist-db to work with <oXygen/>

### ISSUE THREAD

**Issue by djbpitt** [eXist-db/exist#1549]
### What is the problem

Documentation at https://exist-db.org/exist/apps/doc/oxygen.xml does not enable connection to a running eXist-db instance.

### What did you expect

I expected the instructions to explain fully how to configure &lt;oXygen/&gt; to connect to eXist-db.

### Describe how to reproduce or add a test

Following the instructions at the specified page and doing nothing else fails to connect to the eXist instance.

### Fix

Recursively adding all jar files in the entire eXist installation fixes the problem. This is not a wholly satisfactory solution because it's too brute-force. A satisfactory solution would add only the jar files that are actually needed, but I don't know enough to be able to identify them myself.

### Context information

- eXist-db version: 3.4.1 official release
- Java version: java version "1.8.0_45", Java(TM) SE Runtime Environment (build 1.8.0_45-b14)
- Operating system (MacOs El Capitan 10.11.6)
- 64 bit
- No custom changes in e.g. conf.xml

**Comment by joewiz** [comment]
Which version of oxygen had this problem?

**Comment by djbpitt** [comment]
<oXygen/> XML Editor 19.0, build 2017062918

**Comment by duncdrum** [comment]
I actually think we should link to oxygen's documentation. From quick browsing everything after Step 5 doesn't seem to be necessary anymore. Simply selecting exist-db in the drop down usually does the trick for me. @djbpitt what was the error you were getting? Did you try to connect via ssl? to localhost?

**Comment by djbpitt** [comment]
The first oddity that I noticed as that there is no lib/core/ws-commons-1.0.2.jar, which is one of the files mentioned in the documentation. I'm using a localhost installation of eXist as the "data source" (in the &lt;oXygen/&gt; configuration) to connect to two different instances of eXist 3.4.1, one on localhost and the other on a remote server. I was unable to connect to either until I tried the brute-force addition of all available jars in the entire eXist installation, although I suspect that only one or a few of those may have been needed—perhaps to replace the one mentioned in the documentation that is no longer there. I'm not using ssl.

**Comment by duncdrum** [comment]
Filling in: ``host, port, user, password`` on ``Preferences > Data sources > Create exist-db XML connection``  is usually all that is required, with Oxygen copying jars automatically.  So yes, the docs appear outdated.

@joewiz i think its safe to add a label to the issue. Somebody who can go through the steps on a fresh system might be able to update the docs or raise an issue with the oxygen folks? 

@djbpitt My systems has almost identical specs to yours,  each has 7 exist instances in oxygen, and it always worked on first attempt, without the need for any user intervention.  Something is off, but I have no idea what that might be.

**Comment by joewiz** [comment]
As I recall, oXygen's eXist wizard was very new and only available in the newest version of oXygen when the oXygen documentation was last updated. Now that the wizard has been out for several years, I think it's safe to update the directions to use that wizard. I've created a new issue in the documentation repository to track this: https://github.com/eXist-db/documentation/issues/131. Please add any additional notes/thoughts there. Thanks for raising this, David!
